diagnostic.md

```diff
@@ -18,58 +21,59 @@ In what file does the above code go?
 In your own words, what does the command `bundle exec rake db:migrate` do?
 
 ```md
-// your response here
+/running db:migrate runs the change or up method for all the migrations that haven't been run.
```

Comment:
`bundle exec rake db:migrate` executes all of the database migration files in `db/migrate` and generates a new database schema file, `db/schema.rb`.